tests/behat/features/fields.feature

```diff
@@ -173,6 +173,18 @@ Feature: Fields for Publication content type
 
     And I should see text matching "Social sharing"
 
-    And I see field "Show content rating?"
+    And I should see text matching "Content components"
```

Comment:
@MdNadimHossain these lines will fail behat test for sure.
![image](https://user-images.githubusercontent.com/167788/120130188-e2786500-c208-11eb-9bf5-7ca0a5ff2d5d.png)

Reply:
Thanks @sonnykt 
I have removed it and also added the missing components in the landing page component list.